bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiAccessPointThingHandler.java

```diff
@@ -152,6 +152,10 @@ protected State getChannelState(final UniFiDevice device, final String channelId
                     state = new QuantityType<>(device.getExperience(), Units.PERCENT);
                 }
                 break;
+            case CHANNEL_AP_LED:
+                String override = device.getLedOverride();
+                state = "default".equals(override) ? UnDefType.UNDEF : OnOffType.from("on".equals(override));
+                break;
```

Comment:
Very minor:
```suggestion
                state = "default".equals(override) ? UnDefType.UNDEF : OnOffType.from(override);
```

Reply:
Some how my vscode review plugin is one line off. obvious this comment is about line 157

Reply:
Thanks, indeed that looks better.